docs/sources/panels/field-configuration-options.md

```diff
@@ -83,19 +79,19 @@ Field overrides allow you to change the settings for one field (column in tables
 1. Continue to add overrides to this field by clicking **Add override property**, or you can click **Add override** and select a different field to add overrides to.
 1. When finished, click **Save** to save all panel edits to the dashboard.
 
-## Filter options
+## Override matchers
 
-This section explains all available filter options for field overrides. They are listed in alphabetical order.
+This section explains all available matchers for field overrides. They are listed in alphabetical order.
```

Comment:
Again, "matcher" is not the correct word in this context. They are options or filter options.

Reply:
:+1: we need to find a good explanation/heading for this.  The basic concept is that that there are setting that apply to fields -- the "override" section lets you selectively apply new settings to the results.